app/main.py

```diff
@@ -1,2 +1,61 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+
+    def __init__(self) -> None:
+        self._capacity = 8
+        self._threshold = int(self._capacity * 2 / 3)
+        self._length = 0
+        self._increase_value = 2
+        self.table = [[] for _ in range(self._capacity)]
+
+    def __len__(self) -> int:
+        return self._length
+
+    def __resize_table(self) -> None:
+
+        table_ = self.table.copy()
```

Comment:
Don't use `copy()` here, just assign hash table to a temporary variable.
By the way, is `table` a reserved name? Why `table_`?

Reply:
thank you)